### DISCUSSION TITLE
после quick_start поведение youtube не поменялось

### DISCUSSION THREAD

**Discussion by pavel-v-d** [bol-van/zapret#869] [Q&A]
Подмосковный rt, Tplink Archer C7, OpenWRT, флешка для расширения памяти, zapret v68.

Шел по quick_start, после нескольких blockcheck получилось:

ipv4 rutracker.org curl_test_http : tpws --split-http-req=method --oob
ipv4 rutracker.org curl_test_http : nfqws --dpi-desync=fake --dpi-desync-ttl=5
ipv4 rutracker.org curl_test_https_tls12 : tpws not working
ipv4 rutracker.org curl_test_https_tls12 : nfqws --dpi-desync=fake --dpi-desync-ttl=5

пробовал разные варианты filtering, сайты, которые проверялись при blockcheck, работают, а ютуб нет.

goodbyedpi.exe работает с "-q --native-frag --wrong-chksum --fake-gen 5 --set-ttl 4 --blacklist ..\russia-blacklist.txt --blacklist ..\russia-youtube.txt", но задача чтобы телевизор начал показывать youtube

Подскажите, пожалуйста, в какую сторону смотреть? Увидел "Обход замедления youtube в России #200" - пробовать варианты оттуда?

**Comment by bol-van** [comment]
У ютуба особая блокировка.
Там надо прогонять на GGC серверах. Те, что оканчиваются на googlevideo.com  и видны по F12 в броузере или в шарке по tls.handshake фильтру
С возможным использованием дополнительных параметров
`PKTWS_EXTRA_1="--dpi-desync-fake-tls=/opt/zapret/files/fake/tls_clienthello_www_google_com.bin" PKTWS_EXTRA_2="--dpi-desync-repeats=3" DOMAINS="rr5---sn-n3toxu-axqs.googlevideo.com" ./blockcheck.sh`

Потом делать хостлист и использовать мульти-стратегию

**Comment by pavel-v-d** [comment]
> Потом делать хостлист и использовать мульти-стратегию

попробовал такой вариант - получил новые параметры. но на компьютере youtube открывается, на планшете и телевизоре вижу списки роликов, но видео не воспроизводится.

**Comment by bol-van** [comment]
TLS 1.2 сложнее пробить. Но старье не умеет 1.3

**Comment by pavel-v-d** [comment]
Это я недоработал.
Поставил готовую сборку с v67 - видео запускается.
Основные отличия, в их варианте:
GZIP_LISTS=0
TPWS_SOCKS_ENABLE=0
FLOWOFFLOAD=none
ну и параметры TPWS_OPT, NFQWS_OPT у них другие.

**Comment by planbo1** [comment]
> Подмосковный rt, Tplink Archer C7, OpenWRT, флешка для расширения памяти, zapret v68.
> 
> Шел по quick_start, после нескольких blockcheck получилось:
> 
> ipv4 rutracker.org curl_test_http : tpws --split-http-req=method --oob ipv4 rutracker.org curl_test_http : nfqws --dpi-desync=fake --dpi-desync-ttl=5 ipv4 rutracker.org curl_test_https_tls12 : tpws not working ipv4 rutracker.org curl_test_https_tls12 : nfqws --dpi-desync=fake --dpi-desync-ttl=5
> 
> пробовал разные варианты filtering, сайты, которые проверялись при blockcheck, работают, а ютуб нет.
> 
> goodbyedpi.exe работает с "-q --native-frag --wrong-chksum --fake-gen 5 --set-ttl 4 --blacklist ..\russia-blacklist.txt --blacklist ..\russia-youtube.txt", но задача чтобы телевизор начал показывать youtube
> 
> Подскажите, пожалуйста, в какую сторону смотреть? Увидел "Обход замедления youtube в России #200" - пробовать варианты оттуда?

Совершенно внезапно сегодня настраивал zapret c включенным по умолчанию в OpenWRT, но заблокированном в zapret IP6 и ничего не получалось. Когда разрешил - все запахало. как резолвится youtube.com? По IP4 или IP6?

**Comment by numbereleven-a** [comment]
и так и так

**Comment by pavel-v-d** [comment]
> Совершенно внезапно сегодня настраивал zapret c включенным по умолчанию в OpenWRT, но заблокированном в zapret IP6 и ничего не получалось. Когда разрешил - все запахало. как резолвится youtube.com? По IP4 или IP6?

в zapret я отключал v6, и в коробочном варианте zapret он отключен

**Comment by bol-van** [comment]
В коробочном он отключен, поскольку он достаточно редок среди провайдеров.
Чтобы не тратить лишние ресурсы напрасно.
Но на то и дана голова человеку, чтобы его включить при необходимости.
Никто не знает нужно ли вообще дурить ipv6. Может по ipv6 нет блокировок. Может ipv6 дается через VPN или тоннель брокер, где тоже нет блокировок. Все это выяснять автоматом бессмысленно.

**Comment by planbo1** [comment]
> и так и так

Я бы попробовал на роутере отключить использование IP6 (ну или включить в zapret).

**Comment by planbo1** [comment]
> В коробочном он отключен, поскольку он достаточно редок среди провайдеров. Чтобы не тратить лишние ресурсы напрасно. Но на то и дана голова человеку, чтобы его включить при необходимости. Никто не знает нужно ли вообще дурить ipv6. Может по ipv6 нет блокировок. Может ipv6 дается через VPN или тоннель брокер, где тоже нет блокировок. Все это выяснять автоматом бессмысленно.

Этим инструментом (zapret) пользуются не только прошаренные пользователи. Некоторые просто ставят по инструкции, а стратегии берут "в интернете"/у знакомых. Я (даже не столько я, сколько более опытный товарищ) пытались настроить стратегии, а дело оказалось в ipv6. И как раз из-за того, что он достаточно редок, а в прошивке по-умолчанию он включен, это не сразу стало понятно. Но дурить ipv6 видно нужно, раз без включения zapret и на ipv6 не работало.
Провайдер - МТС, Москва.
